src/main/resources/application.yaml

```diff
@@ -60,13 +75,15 @@ spring:
 
 # Scheduler configuration
 scheduler:
-  camel-route-config: 0 0/120 * * * ?
-  camel-leaf-router-config: 0 */105 * * * ?
+  camel-route-config: 0 51 11 * * ?
+  camel-leaf-router-config: 0 50 11 * * ?
 
```

Comment:
this config need to be updated for prod release to trigger leaf rout at 9 PM and main route at 10 PM daily

Reply:
yes for testing i will 5 mins leaf and 7 mins parent. in att once Testing done will replace 9pm and 10 pm